### ISSUE TITLE
[469b] Need make actual documentation for all voodoo about replication in native side of engine

### ISSUE THREAD

**Issue by SeriousBuggie** [OldUnreal/UnrealTournamentPatches#842]
For example, if someone not know:
If you make any inventory class, scripted replicated (e.g. all rules defined in scripts) work only and only when:
`bNetOwner || bNetInitial`. Or first send data or when someone own this item (in this case replication work only for owner).

Such stuff must be described. Since you can try hours for try figure why replication not work.

Another fancy stuff About Inventory and native replication:
```c++
	if ( bAlwaysRelevant && !bNetInitial ) // only inventory pickups should be like this
	{
			DOREP(Actor,bHidden);
			return Ptr;
	}
```

**Comment by SeriousBuggie** [comment]
It is about modern engine. In UT things can be more weird and not match.
Better use: https://wiki.beyondunreal.com/Everything_you_ever_wanted_to_know_about_replication_(but_were_afraid_to_ask)

Anyway this issue about undocumented hidden voodoo from engine. I spent a lot of time try figure why custom property, defined in script, with proper replication block, not work or work on bNetInitial. And suddenly found this in native code.

I not see any mention about such stuff. 

Also I not remember any explanation for keyword `nativereplication` in class definition.
`class PlayerReplicationInfo expands ReplicationInfo 	native nativereplication;`

All of this must be documented, for reduce pain when try figure why things not work.

**Comment by an-eternity** [comment]
Yes, actual documentation is very missed... Especially such documentation that would consider existing known issues of engine version 1...

For `nativereplication`, IIRC, explanation was saying presence of this specifier means that Class contains variable(s) that are being replicated via the native code... But that explanation either was not quite clear or was incomplete, because it only describes that presence of this specifier means something (hidden in the native part) is being replicated, but does not let you know what exactly (unless you probably have access to that corresponding native code). So, this specifier seem to be practically useless...
